schema.sql

```diff
@@ -2,6 +2,7 @@
 CREATE TABLE animals (
     id SERIAL PRIMARY KEY,
     name VARCHAR(50),
+    species VARCHAR(50),
```

Comment:
@HAMMAS-SALEEM Okay. That is to add statically to show

Reply:
@Lul-Abdifan Yeah you can say that but in real life when you add or remove or update any column in a table then you always use Alter table command because you don't know the size of database and number of tables and it becomes very difficult to manually update the tables so SQL gives this approach of Alter table command to let users or developers to update the database without looking into the tables 🤔

Reply:
@HAMMAS-SALEEM thank you, bro